### PR TITLE
Add libmill.

### LINKED ISSUE #393: Add `Libmill` package
GitHub repo:
- https://github.com/sustrik/libmill

Note: CMake based
## 

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/33051804-add-libmill-package?utm_campaign=plugin&utm_content=tracker%2F1032790&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1032790&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by isaachier**
Resolves #393.

### CODE REVIEW

File: docs/packages/pkg/libmill.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+.. _pkg.libmill:
+
+libmill
+=======
+
+-  `Official <https://github.com/sustrik/libmill>`__
+-  `Hunterized <https://github.com/hunter-packages/libmill>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/libmill/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/libmill.rst]: Pull request number:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

Comment (isaachier) [reply docs/packages/pkg/libmill.rst]: Sure, I also need you to fork the repo please.

Comment (ruslo) [reply docs/packages/pkg/libmill.rst]: Done: https://github.com/hunter-packages/libmill

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2475
* https://travis-ci.org/ingenue/hunter/builds/280572358?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2476
* https://travis-ci.org/ingenue/hunter/builds/280581535?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2481
* https://travis-ci.org/ingenue/hunter/builds/280626926?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.114

**Pull request merged**